deployments/systemd/packages/Makefile

```diff
@@ -45,24 +48,10 @@ tarball:
 		-v $$(pwd)/dist/$(@):/dist \
 		$(IMAGE):$(vVERSION)-$(@)
```

Comment:
We seem to pre-pend the `v` to the VERSION var here. Does this not conflict with the following?

```
PACKAGE_VERSION := $(VERSION:v%=%)
```

Reply:
`vVERSION` is a separate variable defined in the top-level `versions.mk` file and is used when we explicitly want a version string with the leading `v`. In this case, we are using it when constructing the image tag.